lib/model/folder_sendrecv_ownership_unix.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2022 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//go:build !windows
+// +build !windows
+
+package model
```

Comment:
Do we really want to have `folder_<foldertype>_<thing>_<os>.go` ?
Seems like `folder_<foldertype>_<os>.go` would already result in quite a big matrix.